src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -36,10 +34,13 @@ object WorkflowActor {
 case class WorkflowActor(id: WorkflowId,
                          namespace: WdlNamespace,
                          actualInputs: WorkflowCoercedInputs,
-                         backend: Backend) extends LoggingFSM[WorkflowState, WorkflowFailure] {
+                         backend: Backend) extends LoggingFSM[WorkflowState, WorkflowFailure] with WorkflowDescriptor {
+
   implicit val timeout = Timeout(5 seconds)
 
-  private val storeActor = context.actorOf(StoreActor.props(namespace, actualInputs))
+  private val storeActor = context.actorOf(StoreActor.props(namespace, backend.initializeWorkflow(this, actualInputs)))
```

Comment:
FWIW, I think I agree with @geoffjentry.  The initializeWorkflow method really needs a WorkflowDescriptor.  I know that an actor is a workflow descriptor in this case, though it seems cleaner to do it like Jeff suggested.